src/ride/ride.c

```diff
@@ -6049,10 +6049,10 @@ static money32 ride_create(int type, int subType, int flags, int *outRideIndex,
 	ride->music = RCT2_ADDRESS(0x0097D4F4, uint8)[ride->type * 8];
 
 	ride->operation_option = (
-		RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + 4 + (ride->type * 8), uint8) +
-		RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + 4 + (ride->type * 8), uint8) +
-		RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + 4 + (ride->type * 8), uint8) +
-		RCT2_GLOBAL(RCT2_ADDRESS_RIDE_FLAGS + 5 + (ride->type * 8), uint8)
+		RideProperties[ride->type].min_value +
```

Comment:
Any idea what this is doing? Setting default value to something a bit lower than the average of min and max? Can you change it to a x3 to make it easier to read.